msbuild/Xamarin.iOS.Tasks.Windows/Xamarin.iOS.HotRestart.targets

```diff
@@ -217,6 +217,15 @@
 		</Copy>
 	</Target>
 
+	<Target Name="_CreateHotRestartAppMarkers">
+		<MakeDir Directories="$(HotRestartContentStampDir)"/>
+		<MakeDir Directories="@(_HotRestartContentDirContents -> '$(HotRestartContentStampDir)%(DestinationSubDirectory)')"/>
+
+		<Touch AlwaysCreate="true" Files="@(_HotRestartContentDirContents -> '$(HotRestartContentStampDir)%(DestinationSubDirectory)%(FileName)%(Extension).stamp')" />
+		<Touch AlwaysCreate="true" Files="$(HotRestartContentStampDir)$(AssemblyName).hotrestartapp.stamp" />
+		<Touch AlwaysCreate="true" Files="$(HotRestartAppContentDir)$(AssemblyName).hotrestartapp" />
```

Comment:
Could you add comments in the code explaining why these two are needed?

Reply:
> Could you add comments in the code explaining why these two are needed?

Will add them and update the PR

Reply:
@rolfbjarne I've just added some comments. 
Please find here more details on the deployment process (incremental VS full installation) and also on the runtime execution of the Hot Restart app, to understand better about the markers and stamps:

- Deployment: https://github.com/xamarin/XamarinVS/blob/main/src/Tools/Xamarin.iOS.Windows.Client/Installer/ApplicationSession.cs#L83-L110

- Execution: https://github.com/xamarin/XamarinVS/blob/main/src/Tools/Xamarin.iOS.HotRestart.Application/Application.cs#L27-L60